config/routes.rb

```diff
@@ -7,10 +7,12 @@
 
   namespace :account do
     get '/dashboard', to: 'dashboard#index'
-    resources :projects do
-      resources :tasks, except: [:index] do
-        put '/:move', to: 'tasks#move', as: 'move'
-      end
-    end
+    resources :workspaces do
```

Comment:
why do you need 3 nesting?
```ruby
resources :workspaces do
  resources :projects
end

resources :projects do
  resources :tasks, except: [:index] do
    member do
      put :move 
    end
  end
end
```

Reply:
this will make you url helpers shorter (and reduce number of argument is simple_form helpers)